src/components/Input/Input.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC, useState } from "react";
+import { InputType, InputElementProps } from "../Input/type";
+import classes from './Input.module.scss';
+
+const Input: FC<InputElementProps> = ({type, placeHolder, name, required, id, options=[], value}: InputElementProps) => {
+  const [enteredValue, setValue] = useState<string>('');
+  const [isValid, setIsValid] = useState<boolean | null>(true);
+
+  const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newValue = e.target.value;
+    setValue(newValue);
+    setIsValid(validation(type, newValue));
+  }
+
+  return (
+    <div className={classes.inputContainer}>
+      {!isValid? errorElement(type, isValid) : ``}
+      {type === InputType.SELECT &&
+        (<select name={name} className={classes.input} required={required}>
```

Comment:
use `label` tag to wrap up the input
example:|
<img width="529" alt="image" src="https://github.com/BUI-studies/yidlorama/assets/104223028/5b581c98-cc97-470b-9c10-26e541ac1e6b">

Reply:
done